core/server/data/schema/commands.js

```diff
@@ -149,7 +149,7 @@ function createColumnMigration(...migrations) {
         const hasColumn = await conn.schema.hasColumn(table, column);
         const isInCorrectState = dbIsInCorrectState(hasColumn);
 
-        const log = createLog(isInCorrectState ? 'info' : 'warn');
+        const log = createLog(isInCorrectState ? 'warn' : 'info');
```

Comment:
Why was this changed?

Reply:
Was changed as this is the right expected behavior and was a bug in current code. We want to `warn` in migration when they try to run a migration even though its already in correct state, but give them info when its not and migration did its job 😄

Reply:
Oh! Then maybe worth pushing this straight to master ;)

Reply:
Sure, I'll do that as separate.

Reply:
@gargol Also Done 🔥